docs/hardhat-network/reference/README.md

```diff
@@ -124,7 +124,7 @@ networks: {
   hardhat: {
     mining: {
       mempool: {
-        order: "fifo"
+        order: "fifo";
```

Comment:
This `;` is actually incorrect. `prettier` added it.

Reply:
This has bitten me before too! I had an external contributor step in and remove the `;` in some docs I had written. Slightly embarrassing haha.

One option would be to add `<!-- prettier-ignore -->` to the line before ` ```js`, which i found mentioned [here](https://prettier.io/blog/2017/11/07/1.8.0.html).

A seemingly better solution would be to use ` ```json` rather than ` ```js`, which I tried out locally and seems to work as expected.